src/tiff.mli

```diff
@@ -179,6 +208,16 @@ type t
 val ifd : t -> Ifd.t
 (** Access the IFD of the TIFF file *)
 
+val data :
+  t ->
+  File.ro ->
+  ?xoffset:int option ->
```

Comment:
In OCaml, using the `?` makes the value optional so you don't have to say the type is also optional. This makes it a double optional i.e. `int option option`: `?xoffset:int` should suffice :) 

The reason you haven't quite hit this it because in your implementation, you have given the arguments default values of `None` which you don't need to do either. Concretely it can become: 

```ocaml
(* in the mli file *)
val data ... ?xoffset:int -> ....

(* in the ml file *)
let data ?xoffset ?yoffset ... 
```